backend/core-services/egov-workflow-v2/src/main/resources/egov-workflow-v2-persister.yml

```
@@ -273,4 +273,22 @@ serviceMaps:
-      - jsonPath: $.BusinessServices.*.states.*.actions.*.auditDetails.lastModifiedTime
\ No newline at end of file
+      - jsonPath: $.BusinessServices.*.states.*.actions.*.auditDetails.lastModifiedTime
+
+
+  - version: 1.0
+    description: Update ProcessInstance table
+    fromTopic: update-wf-processinstance
+    isTransaction: true
+    queryMaps:
+
+      - query: UPDATE eg_wf_processinstance_v2 SET businessservice=?, status=?, statesla=? WHERE id=?;
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)